generated/java/grpc-google-common-protos/build.gradle

```diff
@@ -6,9 +6,9 @@ buildscript {
 
 apply plugin: 'java'
 
-description = 'GRPC library for grpc-google-common-protos'
+description = 'GRPC library for grpc-google-common-protos (beta)'
```

Comment:
Be consistent - you are using two separate qualifiers (`beta`, `prerelease`)

Reply:
ok, changed to "(prerelease)"